app/src/main/res/layout/fragment_indoor_maps.xml

```diff
@@ -1,58 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<FrameLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:orientation="vertical">
+    android:orientation="vertical"
+    android:background="@android:color/transparent">
 
-    <LinearLayout
+    <ScrollView
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:orientation="vertical"
-        android:background="@color/colorPrimaryLight">
+        android:layout_margin="20dp">
 
-        <com.google.android.material.tabs.TabLayout
-            android:id="@+id/indoor_tabs"
+        <com.davemorrissey.labs.subscaleview.SubsamplingScaleImageView
```

Comment:
Does this handle zooming, or device resolution?

Reply:
I think zooming and extremely large images